most-wanted/modules.md

```diff
@@ -317,3 +317,9 @@ Internationalization and Natural Language Processing
 * ExtUtils::Command or Shell::Command (WIP: [Shell::Command](https://github.com/tadzik/Shell-Command/))
 * App::Ack
 * Email::Filter
+
+## Web Frameworks
+
+* Mojolicious
+* Dancer
```

Comment:
Thanks for your PR !
But perl6 already has a Dancer clone called "Bailador".

See :
https://github.com/ufobat/Bailador
http://perl6maven.com/bailador